frontend/src/features/meeting/lib/formValidation.ts

```diff
@@ -72,7 +72,7 @@ export const validateDuplicateDeparture = (
 
 export const validateForm = (
   departureList: string[],
-  conditionId: string,
+  conditionIds: string[],
```

Comment:
지금 보니 string 타입을 `LocationRequirement`로 수정할 수 있겠네요

```suggestion
import LocationRequirement from '@entities/location/types/LocationRequirement';
conditionIds: LocationRequirement[],
```